chia/wallet/wallet_puzzle_store.py

```diff
@@ -49,16 +49,16 @@ async def create(cls, db_wrapper: DBWrapper2):
             await conn.execute(
                 "CREATE INDEX IF NOT EXISTS derivation_index_index on derivation_paths(derivation_index)"
             )
-
-            await conn.execute("CREATE INDEX IF NOT EXISTS ph on derivation_paths(puzzle_hash)")
-
             await conn.execute("CREATE INDEX IF NOT EXISTS pubkey on derivation_paths(pubkey)")
-
+            await conn.execute("CREATE INDEX IF NOT EXISTS ph on derivation_paths(puzzle_hash)")
             await conn.execute("CREATE INDEX IF NOT EXISTS wallet_type on derivation_paths(wallet_type)")
-
+            # Remove an old, misnamed, redundant index on `wallet_type`
```

Comment:
I think this comment should also include the back story of why it was added and why it was removed and probably a link to neurosis' work